src/player.js

```diff
@@ -547,7 +547,6 @@ export default class Player extends FakeEventTarget {
       if (!this.src) {
         this._prepareVideoElement();
       }
-      this.load();
```

Comment:
This exposes another issue though, if someone calls load and then play we will call load again in play.
@yairans FYI

Reply:
Right, on my case I protect it with engine.src which will be true when _loadPromise exist otherwise we'll call load.

Reply:
`load` is a public api so each middleware must protect itself from multiple calls as we do in our `_load` by `this_src` existence